socorro/processor/breakpad_transform_rules.py

```diff
@@ -353,6 +354,11 @@ def version(self):
 
     #--------------------------------------------------------------------------
     def _execute_external_process(self, command_line, processor_meta):
+        if self.config.get('chatty', False):
```

Comment:
@peterbe I agree however there's already precedent for it in other files in this directory so I am inclined to leave it.